src/lightning/pytorch/loggers/neptune.py

```diff
@@ -34,10 +34,10 @@
 from lightning.pytorch.utilities.model_summary import ModelSummary
 from lightning.pytorch.utilities.rank_zero import rank_zero_only
 
-_NEPTUNE_AVAILABLE = RequirementCache("neptune-client")
+_NEPTUNE_AVAILABLE = RequirementCache("neptune") or RequirementCache("neptune-client")
```

Comment:
For compatibility with 1.0 and pre-1.0

Reply:
Same as above.

Reply:
You can define
```py
_NEPTUNE_LT_1_0 =  RequirementCache("neptune<1.0")
```
if that helps with your logic to make it compatible both ways.

Reply:
I don't think that would be possible as we have new package name from >1.0 onward `neptune` and `neptune-client` corresponds to packages pre 1.0 and post 1.0 for BC compatibility of requirements file.